firewalldb/interface.go

```diff
@@ -141,4 +141,7 @@ type FirewallDBs interface {
 	RulesDB
 	PrivacyMapper
 	ActionDB
+
+	// Close closes the underlying store.
+	Close() error
```

Comment:
dont think we need this

Reply:
just call t.Cleanup sooner

Reply:
(again commit message could explain more as it took me a sec to figure out why you're adding this)

Reply:
Thanks :pray:! Addressed with a new commit in the latest push :)